src/wfqueue.ts

```diff
@@ -124,8 +124,10 @@ class WFQueueRunner
         const logger = exec.logger;
         logger.info("Workflow queues:");
         for (const [qn, q] of this.wfQueuesByName) {
-            const conc = q.concurrency !== undefined ? `${q.concurrency}` : 'No concurrency limit set';
+            const conc = q.concurrency !== undefined ? `global concurrency limit: ${q.concurrency}` : 'No concurrency limit set';
             logger.info(`    ${qn}: ${conc}`);
+            const workerconc = q.workerConcurrency !== undefined ? `worker concurrency limit: ${q.workerConcurrency}` : 'No worker concurrency limit set';
+            logger.info(`    ${qn}: ${workerconc}`);
```

Comment:
@apoliakov btw I noticed these logs are not in order in the dashboard. I assume due to timestamps granularity.

<img width="683" alt="Screenshot 2025-01-29 at 22 08 52" src="https://github.com/user-attachments/assets/be6af399-61da-4f16-8a14-8464fc474006" />